packages/typegpu/src/index.ts

```diff
@@ -11,8 +12,9 @@ export const tgpu = {
   Storage,
   Vertex,
 
-  createBuffer,
+  bindGroupLayout,
 
```

Comment:
Is the whitespace intentionally there for separation?

Reply:
Yess, I wanted to separate the actually doing something methods, the resource-creating methods and the legacy methods.